### PR TITLE
DDCNL-2427

### PR DESCRIPTION
**Pull request by oc-ste**
- [x] Unit tests passing
- [x] Coverage reached
- [x] Acceptance tests passing
- [x] Reviewed

### CODE REVIEW

**Review by shikkhasingh — APPROVED** [review]

File: test/views/html/IncomeSourceSummaryViewSpec.scala [hunk 1]
```diff
@@ -126,8 +126,26 @@ class IncomeSourceSummaryViewSpec extends TaiViewSpec {
           rtiAvailable = false)
 
         val doc = Jsoup.parse(views.html.IncomeSourceSummary(model).toString())
-        doc must haveHeadingH2WithText(messages("tai.income.details.incomeReceivedToDate"))
         doc must haveSpanWithText(messages("tai.rti.down"))
+        doc must haveSpanWithText(messages("tai.rti.down.updateEmployment"))
+      }
+
+      "rti is unavailable display rti down messages for pensios" in {
```
Comment (shikkhasingh) [inline test/views/html/IncomeSourceSummaryViewSpec.scala:133]: typo in the test name

File: conf/messages [hunk 2]
```diff
@@ -1400,5 +1400,7 @@ tai.unauthorised.message = You will have to sign in again. If you were updating
 tai.unauthorised.button-text = Sign in
 
 tai.rti.down= We cannot show this information at this time. Try again later to view income received to date.
-tai.rti.down.ceased_employments = We cannot show this information at this time. Try again later to view employments that have ended.
```
Comment (shikkhasingh) [inline conf/messages:1403]: Can we please update the references of this message key as well?
I can see two of them inside the code

File: conf/messages.cy [hunk 3]
```diff
@@ -1404,4 +1404,7 @@ tai.unauthorised.message = Bydd yn rhaid i chi fewngofnodi eto. Os oeddech yn di
 tai.unauthorised.button-text = Mewngofnodi
 
 tai.rti.down=Ni allwn ddangos yr wybodaeth hon ar hyn o bryd. Rhowch gynnig arall arni yn nes ymlaen i fwrw golwg dros yr incwm a ddaeth i law hyd yn hyn.
-tai.rti.down.ceased_employments = Ni allwn ddangos yr wybodaeth hon ar hyn o bryd. Rhowch gynnig arall arni yn nes ymlaen i weld y cyflogaethau sydd wedi dod i ben.
+tai.rti.down.ceasedEmployments = Ni allwn ddangos yr wybodaeth hon ar hyn o bryd. Rhowch gynnig arall arni yn nes ymlaen i weld y cyflogaethau sydd wedi dod i ben.
+
+tai.rti.down.updateEmployment=You cannot update or remove this employer at this time. Try again later.
+tai.rti.down.updatePension=You cannot update or remove this pension at this time. Try again later.
```
Comment (shikkhasingh) [inline conf/messages.cy:1410]: welsh translation not available yet ?

**Review by shikkhasingh — CHANGES_REQUESTED** [review]

File: conf/messages.cy (hunk 3 above)
Comment (oc-ste) [reply conf/messages.cy:1410]: Not yet

**Review by shikkhasingh — APPROVED** [review]

**Pull request merged**